builtin/bundle.c

```diff
@@ -42,7 +43,20 @@ static const char * const builtin_bundle_unbundle_usage[] = {
 static int verbose;
 
 static int cmd_bundle_create(int argc, const char **argv, const char *prefix) {
+	int all_progress_implied = 0;
+	int progress = isatty(2);
```

Comment:
You _might_ want to consider making this an `enum` instead of using a magic value `2`...